src/services/fibery/index.js

```diff
@@ -316,7 +316,24 @@ const getTinderGame = async ({ fiberyPublicId }) => {
 };
 
 const getDocument = async fiberyPublicID => {
-  const [document] = await fibery.entity.query(DOCUMENT_VIEW, { $id: fiberyPublicID });
+  const response = await fetch('https://wir.fibery.io/api/views/json-rpc', {
```

Comment:
Maybe add simple wrapper around `fetch`?

Something like simplified version of [this](https://github.com/babajka/babajka-frontend/blob/7b4670d68615497386c5a849f56126376bedec14/utils/request.js#L21)

Reply:
Yeah, I probably should